mhkit/utils/cache.py

```diff
@@ -51,146 +50,101 @@
 def handle_caching(
     hash_params,
     cache_dir,
-    data=None,
-    metadata=None,
-    write_json=None,
+    cache_content=None,
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args